src/types/mount.cr

```diff
@@ -42,20 +48,23 @@ module Docr::Types
 
     @[JSON::Field(key: "Mode")]
     property mode : Int64
+
+    def initialize(@size_bytes, @mode)
+    end
   end
 
   class Mount
     include JSON::Serializable
 
-    @[JSON::Field(key: "Target")]
-    property target : String?
-
     @[JSON::Field(key: "Source")]
     property source : String
 
     @[JSON::Field(key: "Type")]
     property type : String
 
+    @[JSON::Field(key: "Target")]
+    property target : String?
```

Comment:
Clustered and moved the non `?` types to the top of the class, which looks more like I moved `target` down :P